src/libs/nav.js

```diff
@@ -109,7 +109,7 @@ export const updateSearch = (params) => {
 export function PathHashInserter() {
   useOnMount(() => {
     const loc = window.location;
-    const desiredPath = `${process.env.PUBLIC_URL}/`;
+    const desiredPath = '/';
```

Comment:
curious why we are losing the capability to have an ENV var set the url root here?

Reply:
I guess we could use `import.meta.env.BASE_URL` here, but it seems unnecessary. It's never set to anything other than `/`.